elfloader-tool/src/arch-arm/sys_boot.c

```diff
@@ -91,32 +91,8 @@ void main(UNUSED void *arg)
 {
     int num_apps;
 
-#ifdef CONFIG_IMAGE_UIMAGE
```

Comment:
I don't think this functionality should be removed, but a configuration to default to picking the kernel supplied one if it exists.

Reply:
Cool, seems like a better approach. I'll rework the commits to take this approach.